Makefile

```diff
@@ -307,18 +307,28 @@ deploy-minio:
 	$(ECHO) Installing minio
 	$(VECHO) kubectl apply -f minio.yaml
 
-# end-to-tests
+# generic end-to-tests
+.PHONY: prepare-e2e
+prepare-e2e: kuttl start-kind cert-manager install-openshift-routes deploy-minio set-test-image-vars set-test-operator-config build docker-build load-image-operator deploy
+
 .PHONY: e2e
 e2e:
 	$(KUTTL) test
 
-# end-to-tests
+# OpenShift end-to-tests
+.PHONY: prepare-e2e-openshift
+prepare-e2e-openshift: deploy-minio
```

Comment:
any reason why this was moved form the `kuttl-test-openshift.yaml` to the makefile?

Reply:
The original idea was that the kuttl suite will configure the operator.

Reply:
> any reason why this was moved form the `kuttl-test-openshift.yaml` to the makefile?

Yes: when testing downstream products, I do not want to touch the config, and the namespace (tempo-operator-system) doesn't match.

Reply:
imho we have currently two suites:
* e2e: must work on *all platforms*, using either the *community or OpenShift* tempo operator configuration, deployed via *manifests or OLM*
* e2e-openshift: must work on OpenShift, using the OpenShift tempo operator configuration, deployed via OLM

Now, if we want to test the OpenShift suite on e.g. minikube, somewhere _external_ to the test the missing pieces need to be configured: route-controller, cert-manager, opentelemetry-operator, and (probably) replacing the operator config. imho this could/should be in the makefile, but in "fine grained" tasks instead of catch-all `prepare-e2e*` tasks, for example we don't want to configure cert-manager if the operator was deployed via OLM, etc.